registry/storage/driver/s3-aws/s3.go

```diff
@@ -56,16 +56,17 @@ var validRegions = map[string]struct{}{}
 
 //DriverParameters A struct that encapsulates all of the driver parameters after all values have been set
 type DriverParameters struct {
-	AccessKey     string
-	SecretKey     string
-	Bucket        string
-	Region        string
-	Encrypt       bool
-	Secure        bool
-	ChunkSize     int64
-	RootDirectory string
-	StorageClass  string
-	UserAgent     string
+	AccessKey      string
+	SecretKey      string
+	Bucket         string
+	Region         string
+	Encrypt        bool
+	Secure         bool
+	ChunkSize      int64
+	RootDirectory  string
+	StorageClass   string
+	UserAgent      string
+	RegionEndpoint string
```

Comment:
This is a minor nit, but could you move RegionEndpoint to just after Region in this struct?